code/game/objects/items/devices/instruments.dm

```diff
@@ -302,5 +302,8 @@
 	var/choice = input(M,"What instrument would you like to order?","Jazz Express") as null|anything in display_names
 	if(!choice || !M.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
 		return
-	var/instrument = new display_names[choice]
+	var/willruntimeifelse = display_names[choice]
+	var/instrument = new willruntimeifelse()

```

Comment:
do you really need 2 variables for this?

Reply:
willruntimeifelse

Reply:
To me that just looks like you're tripping the system since that isn't fixing whatever the runtime is but if it works 🤷‍♀️